services/eventsapi/main.py

```diff
@@ -69,4 +74,11 @@ async def create_event(event: Event):
                 Body=event.json()
             )
     else:
-        print(f"Received event: {event.json()}")
\ No newline at end of file
+        print(f"Received event: {event.json()}")
+
+    if KAFKA_BROKERS:
+        producer = KafkaProducer( bootstrap_servers=KAFKA_SERVER, api_version=(2, 0, 2))
+        producer.send(KAFKA_TOPIC, event.json().encode('utf-8'))
+
+        producer.close()
+        print(f"Received event: {event.json()}")
```

Comment:
I believe this `print()` is redundant relative to the one above on L77 when S3 is not enabled (which is the typical case) and may not be necessary anyway since on this codepath we can check `redpanda` to see if events show up there :man_shrugging: .